executor_test.go

```diff
@@ -600,8 +600,8 @@ func TestExecutor_Execute_TopN_Attr_Src(t *testing.T) {
 	}
 }
 
-// Ensure a Sum() query can be executed.
-func TestExecutor_Execute_Sum(t *testing.T) {
+// Ensure a SumReduce() query can be executed.
```

Comment:
Should we change this test name and comment back to `Sum` instead of `SumReduce`?